static/bidder-params/bidtheatre.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Bidtheatre Adapter Params",
+  "description": "A schema which validates params accepted by the Bidtheatre adapter",
+  "type": "object",
+  "properties": {
+    "publisherId": {
+      "type": "string",
+      "description": "Publisher ID",
+      "minLength": 36,
+      "maxLength": 36
```

Comment:
I noticed 36 was used as the request number. Could you explain the reasoning? Just want to ensure I fully understand

Reply:
We will require publishers to be approved before we will bid on their inventory via Prebid. Upon approval the publisher will be assigned an UUID , this UUID is a string and always 36 characters long and at this time the only required bid param.